packages/maskbook/src/utils/hooks/useMenu.tsx

```diff
@@ -14,8 +14,13 @@ export function useMenu(...menus: (JSX.Element | undefined)[]) {
             {menus}
         </ShadowRootMenu>,
         useCallback((anchorElOrEvent: HTMLElement | { currentTarget: HTMLElement }) => {
-            if (anchorElOrEvent instanceof HTMLElement) anchorElRef.current = anchorElOrEvent
-            else anchorElRef.current = anchorElOrEvent.currentTarget
+            let element: HTMLElement
+            if (anchorElOrEvent instanceof HTMLElement) {
+                element = anchorElOrEvent
+            } else {
+                element = anchorElOrEvent.currentTarget
+            }
+            anchorElRef.current = anchorSibling ? (element.nextElementSibling as HTMLElement) ?? undefined : element
```

Comment:
Set the anchorEl with currentTarget's bottom sibling, otherwise the token name would be closed over.

<img src="https://user-images.githubusercontent.com/63733714/108081094-8a7b7f80-70ab-11eb-94b3-cbaf7207e253.png" width=300 />

Reply:
please add it into the code as comment thanks